support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -77,12 +77,9 @@ Resources:
               Action:
               - sqs:GetQueueUrl
               - sqs:SendMessage
-              Resource: arn:aws:sqs:eu-west-1:865473395570:contributions-thanks
-            - Effect: Allow
-              Action:
-              - sqs:GetQueueUrl
-              - sqs:SendMessage
-              Resource: arn:aws:sqs:eu-west-1:865473395570:contributions-thanks-dev
```

Comment:
Is this not necessary any more?

Reply:
Ah, I see what I missed: there used to be two queues here and now there’s only one. We still have the Allow for the queue.